lib/UnhandledSchemeError.js

```diff
@@ -0,0 +1,33 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Ivan Kopeykin @vankop
```

Comment:
this is not correct

Reply:
I renamed it from your `UnsupportedSchemeError`